micro-ui/web/micro-ui-internals/packages/modules/core/src/components/RoleBasedEmployeeHome.js

```
@@ -139,29 +175,60 @@ export const RoleBasedEmployeeHome = ({ modules, additionalComponent }) => {
+    const howItWorks = getLinkByType(moduleData, "howItWorksLink");
+    const userManual = getLinkByType(moduleData, "userManualLink");
+
     const propsForModuleCard = {
       icon: "SupervisorAccount",
       moduleName: t(moduleData?.label),
       metrics: [],
-      links: Digit.Utils.getMultiRootTenant() ? moduleData.links?.filter((item) => item.label !== "Configure") : moduleData.links,
-      centreChildren: [<div>{t(Digit.Utils.locale.getTransformedLocale(`MODULE_CARD_DESC_${current}`))}</div>],
+      links: Digit.Utils.getMultiRootTenant() ? getFilteredLinks(moduleData) : moduleData.links,
+      centreChildren: [
+        <div>{t(Digit.Utils.locale.getTransformedLocale(`MODULE_CARD_DESC_${current}`))}</div>,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)